ClientMultiMapLockTest.testLockTTLExpires_thenOtherThreadCanObtain

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.client.multimap.ClientMultiMapLockTest.testLockTTLExpires_thenOtherThreadCanObtain(ClientMultiMapLockTest.java:358)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast-client/160/testReport/junit/com.hazelcast.client.multimap/ClientMultiMapLockTest/testLockTTLExpires_thenOtherThreadCanObtain/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.client.multimap.ClientMultiMapLockTest.testLockTTLExpires_thenOtherThreadCanObtain(ClientMultiMapLockTest.java:358)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast-client/200/testReport/junit/com.hazelcast.client.multimap/ClientMultiMapLockTest/testLockTTLExpires_thenOtherThreadCanObtain/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.client.multimap.ClientMultiMapLockTest.testLockTTLExpires_thenOtherThreadCanObtain(ClientMultiMapLockTest.java:358)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast-client/220/testReport/junit/com.hazelcast.client.multimap/ClientMultiMapLockTest/testLockTTLExpires_thenOtherThreadCanObtain/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:782)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:775)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:767)
	at com.hazelcast.client.multimap.ClientMultiMapLockTest.testLockTTLExpires_thenOtherThreadCanObtain(ClientMultiMapLockTest.java:358)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast-client/927/testReport/junit/com.hazelcast.client.multimap/ClientMultiMapLockTest/testLockTTLExpires_thenOtherThreadCanObtain/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/multimap/ClientMultiMapLockTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/multimap/ClientMultiMapLockTest.java
@@ -347,7 +347,7 @@ public class ClientMultiMapLockTest extends HazelcastTestSupport {
         new Thread() {
             public void run() {
                 try {
-                    if (mm.tryLock(key, 4, TimeUnit.SECONDS)) {
+                    if (mm.tryLock(key, 10, TimeUnit.SECONDS)) {
                         tryLockSuccess.countDown();
                     }
                 } catch (InterruptedException e) {

